packages/gatsby-theme-carbon/src/components/LeftNav/LeftNav.module.scss

```diff
@@ -1,20 +1,87 @@
+// White theme
```

Comment:
Great job nuking those nasty overrides in this file. Is it possible to increase the specificity through classes/more specific selectors than using important?